### PR TITLE
Drag and drop

### PR DESCRIPTION
**Pull request by tiwiz**
Add Drag&Drop feature to feed title in both For You and Saved fields

### CODE REVIEW

**Comment by riggaroo** [comment]
Possible to add a screen recording?

File: app/.gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
-/build
\ No newline at end of file
+/build
+/.idea/.gitignore
+/.idea/gradle.xml
+/.idea/misc.xml
+/.idea/vcs.xml
```
Comment (SimonMarquis) [inline app/.gitignore:5]: This is not an expected change, please discard it.

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 accompanist = "0.34.0"
 androidDesugarJdkLibs = "2.0.4"
 # AGP and tools should be updated together
-androidGradlePlugin = "8.4.0"
+androidGradlePlugin = "8.5.1"
```
Comment (SimonMarquis) [inline gradle/libs.versions.toml:5]: Does your change need this?
If yes, we should update it in a distinct review.
If no, discard this change.

Comment (SimonMarquis) [reply gradle/libs.versions.toml:5]: - https://github.com/android/nowinandroid/pull/1536

Comment (tiwiz) [reply gradle/libs.versions.toml]: Discarded

**Comment by tiwiz** [comment]
Closing PR - will open another one with only drag

**Pull request closed without merging**